pom.xml

```diff
@@ -16,7 +16,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.diffblue.cover</groupId>
   <artifactId>cover-annotations</artifactId>
-  <version>1.2.0</version>
+  <version>1.3.0</version>
```

Comment:
@ThomasPerkins1123 from process point of view, are you happy for this commit to be part of the same PR or should it be a standalone PR?

Reply:
Yes I am happy with this on this repo, as long as its always its own commit